### ISSUE TITLE
`for` loop misbehaving when using `continue`

### ISSUE THREAD

**Issue by plajjan** [actonlang/acton#1265]
### Acton Version

0.15.0.20230505.15.18.40

### Steps to Reproduce

```
actor main(env):
    for i in range(0, 10, 1):
        print(i)
        continue

    await async env.exit(0)
```

### Expected Behavior

Print numbers 0 to 10 and then quit.

### Actual Behavior

Infinite loop just printing `0`over and over...

**Comment by plajjan** [comment]
This feels like it's the same or at least related to #1194.

@nordlander I think we should prioritize these.

**Comment by nordlander** [comment]
Fixed by #1491.
